uui/components/dnd/DropMarker.tsx

```diff
@@ -3,7 +3,11 @@ import cx from 'classnames';
 import { DndActorRenderParams, IHasCX } from '@epam/uui-core';
 import css from './DropMarker.module.scss';
 
-export interface DropMarkerProps extends DndActorRenderParams, IHasCX {
+interface DropMarkerProps extends DndActorRenderParams, IHasCX {
+    /**
+     * Pass 'true' to enable Blocker
+     * If false, nothing will be rendered
```

Comment:
Let's remove 'If false, nothing will be rendered'. Sorry, it was my mistake in previous suggestion.